websites/S/SOAP2DAY/dist/metadata.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.3",
+  "service": "SOAP2DAY",
+  "author": {
+    "name": "fun",
+    "id": "217348698294714370"
+  },
+  "category": "socials",
```

Comment:
I don't think it's social at all.

Reply:
This was meant to be "stream" or what have you.